applications/IgaApplication/custom_processes/output_quadrature_domain_process.h

```diff
@@ -0,0 +1,132 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+#if !defined(KRATOS_OUTPUT_QUADRATURE_DOMAIN_PROCESS_H_INCLUDED )
+#define  KRATOS_OUTPUT_QUADRATURE_DOMAIN_PROCESS_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+
+#include "processes/process.h"
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+/**
+ * @class OutputQuadratureDomainProcess
+ * @ingroup KratosCore
+ * @brief This class is used in order to check results using a json file containing the solution a given model part with a certain frequency
+ * @details This stores the dababase in a class denominated ResultDatabase which considers Table to store the information, therefore being able to interpolate results
+ * @author Vicente Mataix Ferrandiz
```

Comment:
Shouldn't this be your name? Furthermore, I don't completely understand the description.

Reply:
And my final comment is that you should check Codacy.